example_client/benchmark/Dockerfile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM haproxy:2.3.10
+
+RUN apt-get update
+RUN apt-get upgrade -y
+RUN apt-get install -y python3
+RUN apt-get install -y python3-pip
+RUN apt-get install -y curl
+
+RUN curl https://bootstrap.pypa.io/get-pip.py | python3
+
+RUN python3 -m pip install numpy
+RUN python3 -m pip install grpcio
+RUN python3 -m pip install requests
+RUN python3 -m pip install tensorflow==2.*
+RUN python3 -m pip install tensorflow-serving-api
+RUN python3 -m pip install pymongo
+RUN python3 -m pip install retry
+RUN python3 -m pip install PyYAML
+RUN python3 -m pip install pypng
+RUN python3 -m pip install nvidia-pyindex
+RUN python3 -m pip install tritonclient[all]
```

Comment:
Is it possible to use benchmark client without docker?
Maybe we could extract pip packages to requirements.txt, so it would be possible to use without having to build the docker image?

Reply:
it is possible but not recommended because in docker we support prometheus service